test/bigtest/network/config.js

```diff
@@ -22,6 +22,10 @@ export default function config() {
     // };
   });
 
+  this.put(`${ORDERS_API}/:id`, (schema, request) => {
+    return schema.orders.find(request.params.id).attrs;
```

Comment:
I'm not sure that okapi back-end works this way and returns updated data on PUT requests. Do you think it's OK to have a different behavior here, in Mirage config?

Reply:
I think it returns, but I'll check

Reply:
yeah, it doesn't return anything